pkg/application/app.go

```diff
@@ -134,7 +134,7 @@ func (app *Application) Validate() error {
 		return errors.New("app is nil")
 	}
 	if app.Name == "" {
-		return errors.New("name is required")
+		return errors.New("please provide an existed App name")
```

Comment:
What does this mean?

Reply:
I intended to provide a more specific error message for `vela app run`.